### PR TITLE
Improve buy form

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @cbernat, @fcoronelmakingsense and also @GusBaamonde and @damianmuti (because I am making some minor changes in HTML structure),

I was working on another task, and these changes come alone, I tried to split each different improvement in a different commit. 

Could you review?

### CODE REVIEW

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { InjectAppServices } from '../../services/pure-di';
 
+const fieldNames = {
+  selectedPlanId: 'selectedPlanId',
+  message: 'message',
```
Comment (mvillaseco) [inline src/components/UpgradePlanForm/UpgradePlanForm.js]: extra ,

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 2]
```diff
@@ -52,31 +66,33 @@ class UpgradePlanForm extends React.Component {
...
                   </label>
                   <textarea
                     onChange={this.changeHandler}
-                    value={this.state.userPlanModel.Detail || ''}
-                    name="Detail"
-                    placeholder="Tu mensage"
+                    value={formData[fieldNames.message] || ''}
+                    id={fieldNames.message}
+                    name={fieldNames.message}
+                    placeholder="Tu mensage" // TODO: translate it
```
Comment (andresmoschini) [inline src/components/UpgradePlanForm/UpgradePlanForm.js]: Oh, I will try to fix it before merging.

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 3]
```diff
@@ -89,9 +105,10 @@ class UpgradePlanForm extends React.Component {
                 <FormattedMessage id="common.cancel" />
               </button>
               <button
+                type="submit"
```
Comment (andresmoschini) [inline src/components/UpgradePlanForm/UpgradePlanForm.js]: I will also apply this before merging:

image

File: src/components/UpgradePlanForm/UpgradePlanForm.js (hunk 2 above)
Comment (andresmoschini) [reply src/components/UpgradePlanForm/UpgradePlanForm.js]: Ready.

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 4]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { InjectAppServices } from '../../services/pure-di';
 
+const fieldNames = {
```
Comment (andresmoschini) [inline src/components/UpgradePlanForm/UpgradePlanForm.js]: What do you think about it @fcoronelmakingsense? should I keep these constants for the names? or should I remove them?

Comment (andresmoschini) [reply src/components/UpgradePlanForm/UpgradePlanForm.js]: Any other consideration before merging?

Comment (cbernat) [reply src/components/UpgradePlanForm/UpgradePlanForm.js]: If we are going to use this type of consts for handling string constants in everywhere, we should agree on this, there are a lot of constants strings in doppler. In angularJS we used to have a constants file, here I'm not so sure of the best way to handle this. Perhaps we could inject constants? Keep in mind we will see this types again in the future surely @andresmoschini .

Comment (cbernat) [reply src/components/UpgradePlanForm/UpgradePlanForm.js]: I understand this constants are only for this model, ignore my previous comment , I will remove it. Will we handle string constants this way from now on? @fcoronelmakingsense @andresmoschini

**Pull request merged**